### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MorwennaG**
I've done an edit of the Readme to try to make it a little easier to digest and bring some of the more important user information to the forefront. I've also added a little Let me know your thoughts!

### Contribution checklist
 - [ ] Tests have been added for boss room and/or utilities pack
 - [ ] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] JIRA ticket ID is in the PR title or at least one commit message
 - [ ] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,144 @@
...
+
+
+__PLEASE NOTE__:
+Boss Room is compatible with the latest Unity Long Term Support (LTS) editor version, currently [2021 LTS](https://unity.com/releases/2021-lts). Please include standalone support for Windows/Mac in your installation. <br>
+
+**Boss Room has been developed and tested on the following platforms:**
+- Windows
+- Mac
+- iOS
+- Android <br>
```
Comment (SamuelBellomo) [inline README.md:14]:
```suggestion
Boss Room is a fully functional co-op multiplayer RPG made with Unity Netcode. It is an educational sample designed to showcase typical gameplay [patterns](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom/bossroom-actions/index.html) that are frequently featured in similar networked games.


__PLEASE NOTE__:
Boss Room is compatible with the latest Unity Long Term Support (LTS) editor version, currently [2021 LTS](https://unity.com/releases/2021-lts). Please include standalone support for Windows/Mac in your installation. <br>

**Boss Room has been developed and tested on the following platforms:**
- Windows
- Mac
- iOS
- Android <br>
```

File: README.md [hunk 2]
```diff
@@ -1,58 +1,144 @@
 ![Banner](Documentation/Images/Banner.png)
 ## Co-op multiplayer RPG and utilities built with Unity Netcode for GameObjects
 
-|[Feedback Form](#feedback-form) |
-| -- |
+Boss Room is a fully functional co-op multiplayer RPG made with Unity Netcode. It is an educational sample designed to showcase typical gameplay [patterns](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom/bossroom-actions/index.html) that are frequently featured in similar networked games.
```
Comment (SamuelBellomo) [inline README.md:4]:
```suggestion
Boss Room is a fully functional co-op multiplayer RPG made with Unity Netcode. It is an educational sample designed to showcase typical gameplay [patterns](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom/bossroom-actions/index.html) that are frequently featured in similar networked games. fdsaasdf
```

Comment (SamuelBellomo) [inline README.md:4]:
```suggestion
Boss Room is a fully functional co-op multiplayer RPG made with Unity Netcode. It is an educational sample designed to showcase typical gameplay [patterns](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom/bossroom-actions/index.html) that are frequently featured in similar networked games. qweewq
```

File: README.md [hunk 3]
```diff
@@ -1,58 +1,144 @@
...
+**Boss Room has been developed and tested on the following platforms:**
+- Windows
+- Mac
+- iOS
+- Android <br>
 
-| Support is available on [Discord](https://discord.gg/mNgM2XRDpb) and [forums](https://forum.unity.com/forums/multiplayer.26/) to help you work through issues you may encounter when using Boss Room. |
+**Boss Room has been tested for use on the following devices:**
+- iPhone 6S
+- Pixel 3 <br>
```
Comment (SamuelBellomo) [inline README.md:18]: @jilfranco-unity what's the device you're testing with again? That's the J3 right?

File: README.md [hunk 4]
```diff
@@ -117,104 +203,50 @@ See [ART_NOTES.md](Documentation/ART_NOTES.md) for more information on the art o
 * Scene utils with synced loading screens - [Packages/com.unity.multiplayer.samples.coop/Utilities/SceneManagement/ ](Packages/com.unity.multiplayer.samples.coop/Utilities/SceneManagement/)
 * RNSM custom config - [Packages/com.unity.multiplayer.samples.coop/Utilities/Net/RNSM/CustomNetStatsMonitorConfiguration.asset ](Packages/com.unity.multiplayer.samples.coop/Utilities/Net/RNSM/CustomNetStatsMonitorConfiguration.asset)
 * ParrelSync - [ Packages/manifest.json ](Packages/manifest.json)
+  
+## Troubleshooting
+  
+**Known Issues**   
+ Find an up-to-date list of Boss Room's knows issues in te changelog of our ['Releases'](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/releases) page.
+
```
Comment (SamuelBellomo) [inline README.md:211]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -1,58 +1,144 @@
...
 
-|[Feedback Form](#feedback-form) |
-| -- |
+Boss Room is a fully functional co-op multiplayer RPG made with Unity Netcode. It is an educational sample designed to showcase typical gameplay [patterns](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom/bossroom-actions/index.html) that are frequently featured in similar networked games.
+
+
+__PLEASE NOTE__:
+Boss Room is compatible with the latest Unity Long Term Support (LTS) editor version, currently [2021 LTS](https://unity.com/releases/2021-lts). Please include standalone support for Windows/Mac in your installation. <br>
+
+**Boss Room has been developed and tested on the following platforms:**
```
Comment (chrispope) [inline README.md:10]: Did we test against console platforms for this release?

File: README.md [hunk 6]
```diff
@@ -1,58 +1,144 @@
...
+- Android <br>
 
-| Support is available on [Discord](https://discord.gg/mNgM2XRDpb) and [forums](https://forum.unity.com/forums/multiplayer.26/) to help you work through issues you may encounter when using Boss Room. |
+**Boss Room has been tested for use on the following devices:**
+- iPhone 6S
+- Pixel 3 <br>
+
+Find our full project documentation, [here](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom/bossroom/index.html).
+
+To connect and find support, join the multiplayer community on the Unity [Discord](https://discord.gg/mNgM2XRDpb) and [Forum](https://forum.unity.com/forums/multiplayer.26/).<br> Alternatively, tell us what you think using our new [Feedback Form](#feedback-form) <center>.
```
Comment (chrispope) [inline README.md:22]: I don't want the feedback form framed as an alternative to Discord/Forums. 
Discord/Forums should be used for asking technical questions as users go through their learning journeys.

File: README.md [hunk 7]
```diff
@@ -1,58 +1,144 @@
...
+
+## Opening the project for the first time
+
+Once you have downloaded the project, follow the steps below to get up and running:
+ - Check that you have installed the most recent [LTS editor version](https://unity.com/releases/2021-lts).
+ 	- Include standalone support for Windows/Mac in your installation. 
+ - Add the project to the _Unity Hub_ by clicking on the **Add** button and pointing it to the root folder of the downloaded project.
+ 	- __Please note :__ the first time you open the project Unity will import all assets, which will take longer than usual.
+
+ - Once open, select the 'Startup' scene and click the **Play** button. You can then host a new game or join an existing one using the in-game UI.
```
Comment (SamuelBellomo) [inline README.md:93]: This isn't needed anymore. We have an automation that starts the game with Startup first, whatever the scene you're in.

Comment (SamuelBellomo) [inline README.md:93]:
```suggestion
 - Hit the **Play** button. You can then host a new game or join an existing one using the in-game UI.
```

File: README.md [hunk 8]
```diff
@@ -1,58 +1,144 @@
...
 - [Feedback Form](#feedback-form)
 
-## Boss Room
+## Boss Room Overview
 
-Boss Room is a fully functional co-op multiplayer RPG made with Unity Netcode. It is built to serve as an educational sample that showcases certain typical gameplay [patterns](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom-examples/bossroom-actions) that are frequently featured in similar networked games.
+Boss Room is designed to be used in its entirety to help you explore the concepts and patterns behind a multiplayer game flow; such as character abilities, casting animations to hide latency, replicated objects, RPCs, and integration with the [Relay](https://unity.com/products/relay), [Lobby](https://unity.com/products/lobby), and [Authentication](https://unity.com/products/authentication) services.
 
-You can use everything in this project as a starting point or as bits and pieces in your own Unity games. The project is licensed under the Unity Companion License. See [LICENSE.md](LICENSE.md) for more legal information.
+You can use the project as a starting point for your own Unity game or use elements individually.
```
Comment (SamuelBellomo) [inline README.md:57]:
```suggestion
You can use the project as a reference starting point for your own Unity game or use elements individually. The project is licensed under the Unity Companion License. See [LICENSE.md](LICENSE.md) for more legal information.
```
mmm I'd really make it clear this is UCL (and not MIT for example). UCL has some restrictions. I also have a vague memory of legal telling us licences should be referenced in a "reasonably easy place to find" for legal reasons. --> readme was that.
I also changed the way this was worded, I wouldn't reused things in boss room as is, I'd use boss room as a reference. Boss Room wasn't made to be copy-pasted without customization.

File: README.md (hunk 2 above)
Comment (SamuelBellomo) [inline README.md:4]:
```suggestion
Boss Room is a fully functional co-op multiplayer RPG made with Unity Netcode. It is an educational sample designed to showcase typical netcode [patterns](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom/bossroom-actions/index.html) that are frequently featured in similar multiplayer games.
```

File: README.md [hunk 9]
```diff
@@ -1,58 +1,144 @@
...
+  ------------------------------------------
+
+## Exploring the project
+BossRoom is an eight-player co-op RPG game experience, where players collaborate to fight imps, and then a boss. Players can select between classes that each have skills with didactically interesting networking characteristics. Control model is click-to-move, with skills triggered by a mouse button or hotkey.
+
+One of the eight clients acts as the host/server. That client will use a compositional approach so that its entities have both server and client components.
+
+- The game is server-authoritative, with latency-masking animations. 
+- Position updates are carried out through NetworkedVars that sync position, rotation and movement speed. 
+- NetworkedVars and Remote Procedure Calls (RPC) endpoints are isolated in a class that is shared between the server and client specialized logic - components.
```
Comment (SamuelBellomo) [inline README.md:106]:
```suggestion
- The game is server-authoritative, with latency-masking animations. 
- Position updates are carried out through NetworkTransform that sync position and rotation. 
```
That's actually not true anymore, suggested correction

File: README.md (hunk 5 above)
Comment (SamuelBellomo) [reply README.md:10]: no

File: README.md [hunk 10]
```diff
@@ -117,104 +203,50 @@ See [ART_NOTES.md](Documentation/ART_NOTES.md) for more information on the art o
...
 * RNSM custom config - [Packages/com.unity.multiplayer.samples.coop/Utilities/Net/RNSM/CustomNetStatsMonitorConfiguration.asset ](Packages/com.unity.multiplayer.samples.coop/Utilities/Net/RNSM/CustomNetStatsMonitorConfiguration.asset)
 * ParrelSync - [ Packages/manifest.json ](Packages/manifest.json)
+  
+## Troubleshooting
+  
+**Known Issues**   
+ Find an up-to-date list of Boss Room's knows issues in te changelog of our ['Releases'](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/releases) page.
+  
+**Bugs**  
+  Report bugs using the [Unity bug submission process](https://unity3d.com/unity/qa/bug-reporting).
```
Comment (SamuelBellomo) [inline README.md:213]:
```suggestion
 Report bugs in Boss Room using Github [issues](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/issues)
 Report NGO bugs using NGO Github [issues](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects)
  Report Unity bugs using the [Unity bug submission process](https://unity3d.com/unity/qa/bug-reporting).
```
Our bug reporting in MTT is very github issues based

File: README.md [hunk 11]
```diff
@@ -117,104 +203,50 @@ See [ART_NOTES.md](Documentation/ART_NOTES.md) for more information on the art o
...
-
----------------
 
 For a deep dive in Unity Netcode and Boss Room, visit our [docs site](https://docs-multiplayer.unity3d.com/).
 
 ## Other samples
 ### Bite-size Samples
-This repository contains a collection of bitesize sample projects and games that showcase different sub-features of NGO. You can review these samples with documentation to understand APIs and features better.
-- [Our various bitesize samples](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize)
+The [Bitesize Samples](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize)  repository is currently being expanded and contains a collection of smaller samples and games, showcasing sub-features of NGO. You can review these samples with documentation to understand our APIs and features better.
```
Comment (SamuelBellomo) [inline README.md:226]:
```suggestion
The [Bitesize Samples](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize)  repository contains a collection of smaller samples and games, showcasing sub-features of NGO. You can review these samples with documentation to understand our APIs and features better.
```
I wouldn't keep transitory statements in a readme like this, it's the best way to forget about those when update the bitesize and not boss room.

File: README.md [hunk 12]
```diff
@@ -1,58 +1,144 @@
...
+## Getting the project
+### Direct download
+ - You can download the latest version of Boss Room from our [Releases](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/releases) page. 
+ - __Alternatively:__ click the green `Code` button and then select the 'Download Zip' option.  Please note that this will download the branch that you are currently viewing on Github.  
+ - __Windows users:__ Using Windows' built-in extraction tool may generate an "Error 0x80010135: Path too long" error window which can invalidate the extraction process. A workaround for this is to shorten the zip file to a single character (eg. "c.zip") and move it to the shortest path on your computer (most often right at C:\\) and retry. If that solution fails, another workaround is to extract the downloaded zip file using [7zip](https://www.7-zip.org/).
+
+
+### Installing Git LFS to clone locally
+
+Boss Room uses Git Large Files Support (LFS) to handle all large assets required locally. See [Git LFS installation options](https://github.com/git-lfs/git-lfs/wiki/Installation) for Windows and Mac instructions.
```
Comment (SamuelBellomo) [inline README.md:79]:
```suggestion
Boss Room uses Git Large Files Support (LFS) to handle all large assets required locally. See [Git LFS installation options](https://github.com/git-lfs/git-lfs/wiki/Installation) for Windows and Mac instructions. This step is only needed if cloning locally. You can also just download the project which will already include large files.
```
Adding a quick precision as this has been confusing users for a while.

File: README.md [hunk 13]
```diff
@@ -77,7 +163,7 @@ See [ART_NOTES.md](Documentation/ART_NOTES.md) for more information on the art o
...
-* NetworkList with custom serialization (LobbyPlayerState) - [Assets/Scripts/Gameplay/GameState/NetworkCharSelection.cs](Assets/Scripts/Gameplay/GameState/NetworkCharSelection.cs)<br>[Assets/Scripts/Gameplay/GameState/NetworkCharSelection.cs](Assets/Scripts/Gameplay/GameState/NetworkCharSelection.cs)
+* NetworkList with custom serialization (LobbyPlayerState) - [Assets/Scripts/Gameplay/GameState/NetworkCharSelection.cs](Assets/Scripts/Gameplay/GameState/NetworkCharSelection.cs)
 * Persistent player (over multiple scenes) - [Assets/Scripts/Gameplay/GameplayObjects/PersistentPlayer.cs ](Assets/Scripts/Gameplay/GameplayObjects/PersistentPlayer.cs)
 * Character logic (including player's avatar) - [Assets/Scripts/Gameplay/GameplayObjects/Character/ ](Assets/Scripts/Gameplay/GameplayObjects/Character/) <br> [ Assets/Scripts/Gameplay/GameplayObjects/Character/ServerCharacter.cs ](Assets/Scripts/Gameplay/GameplayObjects/Character/ServerCharacter.cs)
 * Character movements - [Assets/Scripts/Gameplay/GameplayObjects/Character/ServerCharacterMovement.cs](Assets/Scripts/Gameplay/GameplayObjects/Character/ServerCharacterMovement.cs)
```
Comment (SamuelBellomo) [inline README.md:169]:
```suggestion
* Character movements - [Assets/Scripts/Gameplay/GameplayObjects/Character/ServerCharacterMovement.cs](Assets/Scripts/Gameplay/GameplayObjects/Character/ServerCharacterMovement.cs)
* Client driven movements - Boss Room is server driven with anticipation. See [Client Driven bitesize](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/tree/main/Basic/ClientDriven)
```
This one is not exactly boss room related, but could still help with pointing people in the right direction

File: README.md (hunk 3 above)
Comment (jilfranco-unity) [reply README.md:18]: The J2!

Comment (SamuelBellomo) [inline README.md:18]:
```suggestion
- Samsung Galaxy J2 <br>
```

Comment (SamuelBellomo) [reply README.md:18]: @jilfranco-unity ^^^

Comment (jilfranco-unity) [reply README.md:18]:
```suggestion
- Samsung Galaxy J2 Core <br>
```

File: README.md (hunk 6 above)
Comment (MorwennaG) [reply README.md:22]:
```suggestion
To connect and find support, join the multiplayer community on the Unity [Discord](https://discord.gg/mNgM2XRDpb) and [Forum](https://forum.unity.com/forums/multiplayer.26/).<br> Got feedback? Tell us what you think using our new [Feedback Form](#feedback-form) <center>.
```

File: README.md (hunk 11 above)
Comment (MorwennaG) [reply README.md:224]: I disagree with this one. I think we'll be working on BSS for at least the next year and this is good reminder for folks to keep checking back.

**Review by SamuelBellomo — APPROVED** [review]

**Pull request merged**